src/crimson/net/ProtocolV1.cc

```diff
@@ -774,6 +777,16 @@ seastar::future<> ProtocolV1::read_message()
       ::decode(m.footer, p);
       auto msg = ::decode_message(nullptr, 0, m.header, m.footer,
                                   m.front, m.middle, m.data, nullptr);
+      if (!msg) {
```

Comment:
thanks!
FYI, also noticed this in https://github.com/ceph/ceph/pull/27430/commits/adf6a40efd82a67d5fccbebc8f3dd10696fb2fe8 #815
Seems this implementation doesn't abort the protocol in fault() correctly. Will fix later.